src/Aquifer.Public.API/Endpoints/Resources/Updates/List/Endpoint.cs

```diff
@@ -16,23 +18,70 @@ public override void Configure()
         {
             s.Summary = "Get resource ids that are new or updated since the given UTC timestamp.";
             s.Description =
-                "For a given UTC timestamp and optional language id, get a list of resource ids that are new or have been updated since the provided timestamp. This is intended for users who are storing Aquifer data locally and want to fetch new content.";
+                "For a given UTC timestamp and optional language id, optional ISO language code, or optional collection code get a list of resource ids that are new or have been updated since the provided timestamp. This is intended for users who are storing Aquifer data locally and want to fetch new content.";
```

Comment:
Probably can just say "For a given UTC timestamp, get a list..." at this point, since there are multiple parameters.